mlflow-image/tests/mlrun_example.py

```diff
@@ -0,0 +1,31 @@
+from sklearn.linear_model import LogisticRegression
+import mlflow
+import numpy as np
+import sys
+
+
+def run_example(experiment_name, model_name="test_model"):
```

Comment:
You could add a docstring method documentation to this method which could be helpful to understand the example even better.